push/model/src/test/resources/message-tojson.json

```diff
@@ -0,0 +1,32 @@
+{
+  "ipAddress": null,
+  "clientIdentifier": null,
+  "message": {
+    "alert": "HELLO!",
+    "sound": "default",
+    "badge": 2,
+    "windows": {
```

Comment:
I think on IRC we had a chat about adding specifics, for other platforms as well. @edewit Do you mind starting a thread on the ML for discussion ?